guides/common/modules/proc_registering-capsule-to-satellite-server.adoc

```diff
@@ -16,7 +16,7 @@ For more information on manifests and repositories, see {ContentManagementDocURL
 
 .Proxy and Network Prerequisites
 * The {ProjectServer} base operating system must be able to resolve the host name of the {SmartProxy} base operating system and vice versa.
-* The base operating system on which you want to install {SmartProxyServer} must not be configured to use a proxy to connect to the Red Hat CDN.
+* Ensure there is HTTPS client certificate authentication configured for {SmartProxyServer} to connect to the {ProjectServer}.
```

Comment:
`configured` is wrong since the installer sets up the client certificate authentication. I think it's more like `possible`.

How about something like:

```suggestion
* Ensure HTTPS client certificate authentication is possible between {SmartProxyServer} and {ProjectServer}. HTTP proxies often interfere with this.
```

Reply:
`Ensure` and `Possible` won't work together in a single sentence. I will prepare a new one.